packages/@ama-sdk/core/src/plugins/mock-intercept/mock-intercept.interface.ts

```diff
@@ -17,6 +17,8 @@ export interface MockInterceptRequestParameters {
   disabled: boolean;
   /** Decides if the request should be intercepted or not */
   filter: (request: RequestOptions) => boolean;
+  /** Decides if the operation id should be fetched asynchronously*/
+  asyncOperationId?: boolean;
```

Comment:
Not sure to understand the purpose of this field

Reply:
Removed